components/icon/index.en-US.md

```diff
@@ -0,0 +1,125 @@
+---
+category: Components
+chinese: 图标
+type: Basic
+english: Icon
+toc: false
+---
+
+Meaningful vector graphics, each one can defeat an enemy.
+
+## The specification of naming icons
+
+We provide semantic name for every icon, and naming rules are as follows: 
+
+- Scanning line icon has the similar name with its solid one，but it's distinguished by `-o`, for example, `question-circle`(solid) and `question-circle-o`(scanning line);
```

Comment:
It means what you said. Solid refres to an icon in a full shape, while canning line refers to an icon in an empty shape, maybe they are mistakenly translated. So how should we translate them?